packages/core/crates/core-protocol/src/msg/processor.rs

```diff
@@ -65,21 +67,21 @@ lazy_static::lazy_static! {
     static ref DEFAULT_PRICE_PER_PACKET: U256 = 10000000000000000u128.into();
 }
 
-/// Represents a payload (packet or acknowledgement) at the transport level.
-#[derive(Debug, Clone)]
-pub(crate) struct Payload {
-    pub(crate) remote_peer: PeerId,
-    pub(crate) data: Box<[u8]>,
-}
-
-impl Display for Payload {
-    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        f.debug_struct("Payload")
-            .field("remote_peer", &self.remote_peer)
-            .field("data", &hex::encode(&self.data))
-            .finish()
-    }
-}
+// /// Represents a payload (packet or acknowledgement) at the transport level.
```

Comment:
Should we remove the commented out code?

Reply:
Yes, ty.